### PR TITLE
[python] Add zclwrite command for writing attributes

### LINKED ISSUE #7563: Need zcl write support in python chip device controller
#### Problem
A large amount of test cases in TE3 needs to use zcl write in chip device controller

### PR DESCRIPTION
**Pull request by erjiaqing**
#### Problem
* Missing `zclwrite` for writing attributes
* Fixes #7563

#### Change overview
Add `zclwrite` command to python device controller

#### Testing
- Run `chip-all-cluster-app`
- Run `./src/controller/python/test/test_scripts/mobile-device-test.py -a 172.16.243.197 -t 45`
- The test script is included in cirque.

### CODE REVIEW

**Review by yunhanw-google — APPROVED** [review]

File: src/controller/python/chip-device-ctrl.py [hunk 1]
```diff
@@ -642,6 +642,35 @@ def do_zclread(self, line):
             print("An exception occurred during processing input:")
             print(str(ex))
 
+    def do_zclwrite(self, line):
+        """
+        To read ZCL attribute:
```
Comment (yunhanw-google) [inline src/controller/python/chip-device-ctrl.py:647]: nit, To write

File: src/controller/python/chip-device-ctrl.py [hunk 2]
```diff
@@ -642,6 +642,35 @@ def do_zclread(self, line):
...
+                print('\n'.join(["{}: {}".format(key, cluster_attrs[key]["type"]) for key in cluster_attrs.keys() if cluster_attrs[key].get("writable", False)]))
+            elif len(args) == 6:
+                if args[0] not in all_attrs:
+                    raise exceptions.UnknownCluster(args[0])
+                self.devCtrl.ZCLWriteAttribute(args[0], args[1], int(
+                    args[2]), int(args[3]), int(args[4]), args[5])
+            else:
+                self.do_help("zclwrite")
+        except exceptions.ChipStackException as ex:
+            print("An exception occurred during reading ZCL attribute:")
```
Comment (bzbarsky-apple) [inline src/controller/python/chip-device-ctrl.py:668]:
```suggestion
            print("An exception occurred during writing of ZCL attribute:")
```

**Review by bzbarsky-apple — APPROVED** [review]

File: src/controller/python/chip-device-ctrl.py (hunk 2 above)
Comment (erjiaqing) [reply src/controller/python/chip-device-ctrl.py:668]: Thanks!

File: src/controller/python/test/test_scripts/base.py [hunk 3]
```diff
@@ -141,3 +141,24 @@ def TestReadBasicAttribiutes(self, nodeid: int, endpoint: int, group: int):
             self.logger.exception(f"Following attributes failed: {failed_zcl}")
             return False
         return True
+
+    def TestWriteBasicAttribiutes(self, nodeid: int, endpoint: int, group: int):
```
Comment (Damian-Nordic) [inline src/controller/python/test/test_scripts/base.py:145]: typo: Attributes

File: src/controller/python/test/test_scripts/mobile-device-test.py [hunk 4]
```diff
@@ -61,13 +61,13 @@ def main():
...
               "Failed to finish key exchange")
 
-    logger.info("Testing network commissioning")
-    FailIfNot(test.TestNetworkCommissioning(nodeid=1,
-                                            endpoint=ENDPOINT_ID,
-                                            group=GROUP_ID,
-                                            dataset=TEST_THREAD_NETWORK_DATASET_TLV,
-                                            network_id=TEST_THREAD_NETWORK_ID),
-              "Failed to finish network commissioning")
+    # logger.info("Testing network commissioning")
```
Comment (Damian-Nordic) [inline src/controller/python/test/test_scripts/mobile-device-test.py:64]: Why commented?

**Review by Damian-Nordic — APPROVED** [review]

File: src/controller/python/test/test_scripts/mobile-device-test.py (hunk 4 above)
Comment (erjiaqing) [reply src/controller/python/test/test_scripts/mobile-device-test.py:64]: Sorry, mistakenly commented for local tests

File: src/controller/python/test/test_scripts/base.py (hunk 3 above)
Comment (erjiaqing) [reply src/controller/python/test/test_scripts/base.py]: Thanks

**Comment by woody-apple** [comment]
@erjiaqing Conflict :(

**Review by andy31415 — APPROVED** [review]

**Comment by cjandhyala** [comment]
@erjiaqing does it support writing optional attributes ?

**Comment by bzbarsky-apple** [comment]
>  does it support writing optional attributes ?

I'd expect it to support writing all attributes that are enabled in the controller .zap file...

**Pull request merged**